addons/ofxiOS/src/video/AVFoundationVideoPlayer.m

```diff
@@ -326,7 +326,7 @@ - (BOOL)createAssetReaderWithTimeRange:(CMTimeRange)timeRange {
     }
     
     //------------------------------------------------------------ add audio output.
-    double preferredHardwareSampleRate = [[AVAudioSession sharedInstance] currentHardwareSampleRate];
+    double preferredHardwareSampleRate = [[AVAudioSession sharedInstance] sampleRate];
```

Comment:
ok, i think the first thing we need to do is move the minimum ofxiOS support from the current 5.0 to 6.0. i agree its time to move it up since iOS 9.0 has been released.
we should do this in a separate PR and link it to a relevant issue.
i'll open up the issue now...